Engines/Wine/Verbs/D9VK/script.js

```diff
@@ -11,10 +11,14 @@ include("utils.functions.filesystem.files");
 */
 Wine.prototype.D9VK = function (d9vkVersion) {
     var operatingSystemFetcher = Bean("operatingSystemFetcher");
+    var uiQuestionFactory = Bean("uiQuestionFactory");
     print("NOTE: Wine version should be greater or equal to 3.10");
-    if (operatingSystemFetcher.fetchCurrentOperationSystem() != "Linux")
+    if (operatingSystemFetcher.fetchCurrentOperationSystem().getFullName() !== "Linux")
     {
-        this.wizard().message(tr("D9VK might not work correctly on macOS. This is depending on Metal api support and MoltenVK compatibility layer advancement"));
+        uiQuestionFactory.create(
+				tr("D9VK is currently unsupported on non-Linux operating systems due to MoltenVK implementation being incomplete. Do you want to continue? Chosing yes will skip DXVK verb  installation and continue with other verbs. Chosing no will quit script installation.", winePrefix),
```

Comment:
chosing → choosing, remove duplicate space in "verb   installation", what is `winePrefix` supposed to do?

Reply:
Not sure why it's there, that was in the shortcut reader script.

Reply:
The shortcut reader uses "{0}" in the string. The `tr` will fill this with the variable.

Reply:
So should I remove `winePrefix`?

Reply:
Depends what you want to achieve. You can also insert "${0}" in the string.

Reply:
Not like this. Let me explain
```Javascript
const myVariable = 5;
print(tr("This is my variable: ${0}", myVariable));
```
will print
```
This is my variable: 5
```

Reply:
Ok I guess we dont need that.